lib/mugshot.js

```diff
@@ -276,10 +276,11 @@ function _test(baseName, options, screenshot, done) {
           return done(error);
         }
 
-        done(null, {
-          isEqual: options.acceptFirstBaseline,
-          baseline: baselinePath
-        });
+        if (options.acceptFirstBaseline) {
+          done(null, {isEqual: true, baseline: baselinePath});
+        } else {
+          done(new Error('Baseline missing'));
```

Comment:
@andrei-picus-hs Should I leave it like this or should I return the result object too?